lib/mvi/responses/profile_parser.rb

```diff
@@ -86,6 +86,7 @@ def parse
 
       private
 
+      # rubocop:disable MethodLength
```

Comment:
It might be that, over time, we ended up with the `ProfileParser` and the `MviProfile` inside out, i.e. the parser should be composed into the profile so it can populate its own attributes. Then the only argument to `MviProfile` would be the raw response from MVI.